packages/typespec-test/test/openai/generated/typespec-ts/package.json

```diff
@@ -103,8 +87,30 @@
   },
   "//sampleConfiguration": {
     "productName": "Azure OpenAI API",
-    "productSlugs": ["azure"],
+    "productSlugs": [
+      "azure"
+    ],
     "disableDocsMs": true,
     "apiRefLink": "https://docs.microsoft.com/javascript/api/@msinternal/openai?view=azure-node-preview"
-  }
+  },
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
```

Comment:
Did we want modular exports here such as /api as well?

Reply:
Yeah that's the next iteration. So far I've been working on RLC migration, once I get to modular I'll have those exports